alibi/explainers/tests/utils.py

```diff
@@ -1,8 +1,7 @@
 # flake8: noqa: E731
 # A file containing functions that can be used by multiple tests
-import tensorflow.keras as keras
-
 import numpy as np
+import tensorflow.keras as keras
 
 from keras.utils import to_categorical
```

Comment:
This will use old `Keras`, I think I fixed it in master so rebase/merge should update.

Reply:
yes, I haven't yet merged master.